shell/platform/embedder/embedder.h

```diff
@@ -963,6 +963,43 @@ typedef const FlutterLocale* (*FlutterComputePlatformResolvedLocaleCallback)(
     const FlutterLocale** /* supported_locales*/,
     size_t /* Number of locales*/);
 
+/// Display refers to a graphics hardware system consisting of a framebuffer,
+/// typically a monitor or a screen. This ID is unique per display and is
+/// stable until the Flutter application restarts.
```

Comment:
How would an embedding enforce this? `NSScreen` doesn't have an ID, for instance, AFAICT.

Reply:
This was the display ID I was planning on using: https://developer.apple.com/documentation/coregraphics/cgdirectdisplayid?language=objc.